main.go

```diff
@@ -163,7 +163,9 @@ func main() {
 		}()
 	}
 
-	mgr, err := ctrl.NewManager(ctrl.GetConfigOrDie(), ctrl.Options{
+	restConfig := ctrl.GetConfigOrDie()
+	restConfig.UserAgent = "capi-controller"
```

Comment:
```suggestion
	restConfig.UserAgent = "cluster-api-controller"
```

Reply:
Would it be useful to include CAPI controller version as well?

Reply:
Possibly, yes. It should be included in some build time variables though

Reply:
Should this be kept consistent with what we are using in the `ClusterCacheTracker`?
https://github.com/kubernetes-sigs/cluster-api/blob/master/controllers/remote/cluster.go#L62
Pattern here is something like:
```bash

func buildUserAgent(command, version, sourceName, os, arch, commit string) string {
	return fmt.Sprintf(
		"%s/%s %s (%s/%s) cluster.x-k8s.io/%s", command, version, sourceName, os, arch, commit)
}
```